README.md

```diff
@@ -144,6 +144,12 @@ Sharing in Tomcat.
 
 See instructions [here](http://enable-cors.org/server_tomcat.html) for Tomcat 7 and [here](https://www.dforge.net/2013/09/16/enabling-cors-on-apache-tomcat-6/) for Tomcat 6.
 
+## Usage
+Before using the Order Entry Open Web Application, the administrator should make the following one time configurations using the admin portal. 
+1. Create an Encounter Type called `Order Entry` or an other name.
```

Comment:
Shouldn't you be using a uuid instead of encounter type name?

Reply:
The uuid might vary for different installations when a new Encounter Type for order entry is created.

Reply:
Was using name a suggestion you got from OpenMRS Talk discussions? If yes, can you share the link?

Reply:
https://talk.openmrs.org/t/implementation-of-the-order-entry-ui-application-in-reactjs/16791/39?u=flavia